### ISSUE TITLE
Jupyter and Python 3.7 compatibility

### ISSUE THREAD

**Issue by gusutabopb** [gusutabopb/aioinflux#17]
Currently, the `blocking` mode won't work on Python 3.7 running on Jupyter. The code below:

```python
import aioinflux
c = aioinflux.InfluxDBClient(db='mydb', mode='blocking')
c.show_measurements()
```

Raises `RuntimeError: This event loop is already running`

This is caused by the fact that the latest versions of [Tornado](https://github.com/tornadoweb/tornado) (which is used by Jupyter/ipykernel) runs an asyncio loop on the main thread by default:

```python
# Python 3.7
import asyncio
asyncio.get_event_loop()
# <_UnixSelectorEventLoop running=True closed=False debug=False>

# Python 3.6 (w/ tornado < 5)
import asyncio
asyncio.get_event_loop()
# <_UnixSelectorEventLoop running=False closed=False debug=False>
```

This is being discussed on https://github.com/jupyter/notebook/issues/3397

From an aioinflux perspective, a possible work around would be to start a new event loop on a background thread and use `asyncio.run_coroutine_threadsafe` to run the coroutine and return a `concurrent.futures.Future` object that wraps the result.

**Comment by gusutabopb** [comment]
Another workaround that does not require any code changes is force-downgrading tornado, as mentioned in this comment: https://github.com/jupyter/notebook/issues/3397#issuecomment-418235519

**Comment by gusutabopb** [comment]
This issue has been fixed as of IPython 7.0. For more details, see [this blog post](https://blog.jupyter.org/ipython-7-0-async-repl-a35ce050f7f7).
